api/v1alpha1/stack_type.go

```diff
@@ -4,122 +4,125 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 // +kubebuilder:object:root=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +kubebuilder:resource:path=tinkerbell,scope=Namespaced,categories=tinkerbell,singular=tinkerbell,shortName=tb
+// +kubebuilder:resource:path=stack,scope=Namespaced,categories=stack,singular=stack,shortName=tb
```

Comment:
We should also change `shortName`.

Reply:
Good catch, actually the short name should be removed.